internal/cli/root.go

```diff
@@ -108,6 +108,11 @@ func (c *Client) metalApiConnect(httpClient *http.Client) error {
 	configuration.Debug = checkEnvForDebug()
 	configuration.AddDefaultHeader("X-Auth-Token", c.Token())
 	configuration.UserAgent = fmt.Sprintf("metal-cli/%s %s", c.Version, configuration.UserAgent)
+	configuration.Servers = metal.ServerConfigurations{
+		metal.ServerConfiguration{
+			URL: c.apiURL,
```

Comment:
It appears that this value is empty because the E2E tests are failing with an invalid API URL (it does not contain `https://api.equinix.com/metal/v1` so requests are being made directly to `/path/...`)